requirements.txt

```diff
@@ -1,11 +1,12 @@
-sqlalchemy>=1.1.0
+sqlalchemy[asyncio]>=1.4.24
 python-magic>=0.4.17
 oletools>=0.54
 sdnotify>=0.3.1
-enum34>=1.0.4
 yara-python>=3.6.3
 requests>=2.19.0
 configparser
 pyparsing
```

Comment:
We had a fix for this, where did it go?
Somwhat similar to this: `pyparsing>=2.1.0,<3`

Reply:
It's tracked in #197. We haven't decided yet whether to follow olevba in restricting the version to <3. I think we're procrastinating in hopes olevba raises that limitation.